app/components/works/detail_component.html.erb

```diff
@@ -14,10 +14,10 @@
     </div>
   <% end %>
 
-  <header class="title clearfix"><%# clearfix here prevents the EditButtonCompnent from spilling outside the container if the title is long %>
+  <header class="title clearfix"><%# clearfix here prevents the EditButtonComponent from spilling outside the container if the title is long %>
     <span class="header-text"><%= title %></span>
     <%= helpers.turbo_frame_tag dom_id(work, :edit), src: edit_button_work_path(work), target: '_top' %>
-    <span class="state"><%= render Works::StateDisplayComponent.new(work_version: work_version) %></span>
+    <span class="state"><%= render Works::DetailStateDisplayComponent.new(work_version: work_version) %></span>
```

Comment:
`Works::StateDisplayComponent` is also used on the dashboard and elsewhere, so it can't simply be changed to show this different message for globus wait states.